pkg/status/combinedstatus-resolver.go

```diff
@@ -80,7 +80,8 @@ type CombinedStatusResolver interface {
 	// The returned set contains the identifiers of combinedstatus objects
 	// that should be queued for syncing.
```

Comment:
This comment needs to be updated so that it describes both returned values.

Reply:
Updated.